adapters/bwx/bwx.go

```diff
@@ -0,0 +1,164 @@
+package bwx
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"text/template"
+
+	"github.com/prebid/openrtb/v19/openrtb2"
```

Comment:
We updated the OpenRTB library to version 20. Please merge with the master branch and change this import to `"github.com/prebid/openrtb/v20/openrtb2"`

Reply:
@SyntaxNode done